gem/bin/console

```diff
@@ -1,14 +0,0 @@
-#!/usr/bin/env ruby
-
-require "bundler/setup"
-require "pupperware/pupperware"
-
-# You can add fixtures and/or initialization code here to make experimenting
-# with your gem easier. You can also use a different console, if you like.
-
-# (If you use this, don't forget to add pry to your Gemfile!)
-# require "pry"
-# Pry.start
-
-require "irb"
-IRB.start(__FILE__)
```

Comment:
I don't think we use this IRB file?

Same thing -- we could remove it and make our gem install faster.

Reply:
Removed: https://github.com/puppetlabs/pupperware/pull/170